### ISSUE TITLE
css files not being served

### ISSUE THREAD

**Issue by neilhwatson** [preaction/Statocles#402]
Something changed, might be me, but I can't figure out how to debug it. CSS files are now returned by the daemon as Statocles::Page::File=HASH(0x5443fa8). No CSS content. Using 0.057

**Comment by preaction** [comment]
Is this in the theme directory? Or is this in an application directory? There were some changes to the theme object in 0.057, but the tests pass, so something unexpected is happening.

It's correctly getting the page, but the store is, for some reason, saving the stringification of the page object. The daemon doesn't actually do anything except watch directories for changes (OSX only at the moment) and serve the static site (by default, from the `.statocles/build` directory).

Is this repeatable? You could also try clearing the `.statocles` directory to see if that's somehow a problem (it shouldn't be, as the site is rebuilt always, no intelligence there yet).

If you run `statocles -vv daemon`, you'll get debug output (a single `-v` is info-level logging). It might reveal something, at least whether it's rewriting the right files or not.

Do you happen to have your site on a git repo or something?

**Comment by neilhwatson** [comment]
rm -fr .statocles/build

Problem persists. -vv does not reveal any errors.

This repo: https://github.com/neilhwatson/www.watson-wilson.ca

**Comment by preaction** [comment]
Awesome, thanks. Reproduced. Gonna peel this apart and then mark 0.058

**Comment by preaction** [comment]
So this broke because of the Theme changes in 0.057. The theme itself works fine, and that's what I tested, but the site's use of the theme is broken, and those tests weren't robust enough.

This has been "working for me" because I didn't install the latest version after I released it. Now that I've updated myself, it's broken.

I've got a fix, and will release v0.058 to cpan presently. Thanks for the help tracking this down.
